storages/backends/gcloud.py

```diff
@@ -168,10 +179,15 @@ def _save(self, name, content):
         if CONTENT_TYPE not in blob_params:
             upload_params[CONTENT_TYPE] = file_object.mime_type
 
+        if self.gzip and upload_params.get(CONTENT_TYPE) in self.gzip_content_types:
+            content = self._compress_content(content)
+            file_object.blob.content_encoding = 'gzip'
+            upload_params[CONTENT_ENCODING] = 'gzip'
+
```

Comment:
Can you explain how I should think about `upload_params` vs `blob_params` in the context of this set of changes.